sample/src/main/java/com/airwallex/paymentacceptance/ui/base/BaseMvvmActivity.kt

```diff
@@ -0,0 +1,146 @@
+package com.airwallex.paymentacceptance.ui.base
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.content.res.ColorStateList
+import android.os.Bundle
+import android.util.Log
+import android.view.MenuItem
+import android.widget.ProgressBar
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.ViewModelProvider
+import androidx.viewbinding.ViewBinding
+import com.airwallex.paymentacceptance.R
+import com.airwallex.paymentacceptance.viewmodel.base.BaseViewModel
+import java.lang.reflect.ParameterizedType
+
+abstract class BaseMvvmActivity<VB : ViewBinding, VM : BaseViewModel> : AppCompatActivity() {
```

Comment:
Mvvm is an architecture, maybe we can call it `BasePaymentActivity` instead?

Reply:
ok